objects/obj_enemy_projectile/Step_0.gml

```diff
@@ -2,9 +2,9 @@
 
 y_spd -= grav;
 y -= y_spd;
-
+/*
 if (y > initial_y) {
 	instance_destroy(self);
-}
+}*/
```

Comment:
Do we still need this code that's commented out? If we're not saving it for anything, just get rid of it